api/src/main/java/javax/enterprise/inject/spi/builder/InjectionPointConfigurator.java

```diff
@@ -41,51 +37,6 @@
 public interface InjectionPointConfigurator {
 
     /**
-     * Read the InjectionPoint information from the given {@link Field}.
-     * All relevant information is overwritten.
-     *
-     * @param field defining the InjectionPoint.
-     * @return self
-     */
-    InjectionPointConfigurator read(Field field);
```

Comment:
Hi John, I think that the point of `IPConfigurator` if **not** to give it a field and create an IP point from it (a new one that is). The goal is rather to be able to **temper with currently existing IPs** - meaning changing their qualifiers as there isn't much else you can temper with.

The truth is, we need to also correct the javadoc. It currently says `This API is an helper to configure a new {@link InjectionPoint} instance.`. Which should be changed into something like `This API is a helper to configure an existing {@link InjectionPoint} instance.` Same problem goes for spec - [11.5.7.1](http://docs.jboss.org/cdi/spec/2.0.EDR2/cdi-spec.html#injection_point_configurator). It would be good to include doc/spec text changes i nthis PR as well.